lib/ffi/fasttext.rb

```diff
@@ -72,14 +73,19 @@ module Fasttext
     end
 
     attach_function :create, [:string], :pointer
+    attach_function :create_from_url, [:string], :pointer
     attach_function :destroy, [:pointer], :void
     attach_function :predict_string_free, [:pointer], :void
     attach_function :predict, [:pointer, :string, :int32_t], :strptr
 
     class Predictor
-      def initialize(model_filename)
-        raise "File does not exist" unless ::File.exist?(model_filename) || model_filename.start_with?("http")
-        @ptr = ::FFI::Fasttext.create(model_filename)
+      def initialize(model_name)
+        if is_url?(model_name)
```

Comment:
Think we should remove the `is_url` check and check if file exists here and then loads from url if not, don't think we need both

Reply:
OK, makes sense to me.